packages/cascara/src/ui/Form/modules/ActionEdit/ActionEdit.js

```diff
@@ -14,7 +14,7 @@ const propTypes = {
 const ActionEdit = ({
   cancelLabel = 'Cancel',
   editLabel = 'Edit',
-  saveLabel = 'Save',
+  saveLabel = 'Submit',
```

Comment:
There is a special `ActionEdit` module for `Form` and one for `Table`. These two `ActionEdit` modules work the same except for the `save` button, which for `Form` is a `submit` button. This change in the label is an attempt to provide more semantics.

Reply:
I agree that we should not change the label, I was thinking on semantics and I thought it would make sense for the label to be `Submit` in Forms, since that is the actual action users perform.

Everything has been changed back to `Save`, thanks for the catch! 👍

Reply:
Roger, I agree that this change doesn't make sense over this branch, as you mention, these kind of changes have to be discussed with the rest of the team. Everything has been changed back to its original version. 👍